modules/home-manager/default.nix

```
@@ -1,16 +1,21 @@
-{ config, pkgs, lib, ... }: {
-  imports = [
-    ./alacritty.nix
-    ./bat.nix
-    ./bottom.nix
-    ./btop.nix
-    ./kitty.nix
-    ./starship.nix
-    ./helix.nix
-    ./gtk.nix
-    ./polybar.nix
-    ./tmux.nix
-  ];
+{ config, pkgs, lib, ... }: let
+ extendedLib = import ../lib/mkExtLib.nix lib;
+in {
+  imports = let
+    files = [
+      ./alacritty.nix
+      ./bat.nix
+      ./bottom.nix
+      ./btop.nix
+      ./kitty.nix
+      ./starship.nix
+      ./helix.nix
+      ./gtk.nix
+      ./polybar.nix
+      ./tmux.nix
+    ];
+  in extendedLib.ctp.mapModules config pkgs extendedLib files;
+
   options.catppuccin = {
     flavour = lib.mkOption {
       type = lib.types.enum [ "latte" "frappe" "macchiato" "mocha" ];
```

```suggestion
      type = extendedLib.ctp.types.flavourOption;
```
Do this for the accent option too, Github won't let me suggest changes to lines that aren't in the diff.